packages/map-template/src/components/Map/Map.jsx

```diff
@@ -109,9 +113,39 @@ function Map({ onLocationClick, onVenueChangedOnMap }) {
         return mapsIndoorsInstance.fitVenue(venue).then(() => {
             // Set the map zoom level if the property is provided.
             if (startZoomLevel) {
+                // console.log(startZoomLevel);
                 mapsIndoorsInstance.setZoom(parseInt(startZoomLevel));
             }
+            
+            if (startPitch) {
+                try {
+                    mapsIndoorsInstance.getMap().setPitch(parseInt(startPitch));
+                } catch (error) {
+                    // If setBearing fails, we assume it's a Google map and try to set the heading.
+                    try {
+                        console.log(startPitch);
+                        mapsIndoorsInstance.getMap().setTilt(parseInt(startPitch));
+                    } catch (innerError) {
+                        console.warn("Setting bearing/heading failed for this map instance.");
+                    }
+                }
+            }
+        
+            if (startBearing) {
+                console.log(startBearing);
+                try {
+                    mapsIndoorsInstance.getMap().setBearing(parseInt(startBearing));
+                } catch (error) {
+                    // If setBearing fails, we assume it's a Google map and try to set the heading.
+                    try {
+                        mapsIndoorsInstance.getMap().setHeading(parseInt(startBearing));
+                    } catch (innerError) {
+                        console.warn("Setting bearing/heading failed for this map instance.");
+                    }
+                }
+            }
         });
```

Comment:
Let me know if this is consistent with the map template approach. (see changes in code)

Reply:
Regarding the changes that you have done in the code: 

Both Google Maps and Mapbox MapViews have a tilt and rotate function that we can use, so we are actually able to use those instead. This means that we do not have to check for which map provider is used because we take care of that under the hood in the SDK. 

Therefore you can call the `getMapView()` on the MapsIndoors instance to get a reference of the MapView. Then you can call mapView.tilt() and mapView.rotate().